components/server/src/server2.rs

```diff
@@ -1036,6 +1037,33 @@ where
         if let Some(limiter) = get_io_rate_limiter() {
             limiter.set_low_priority_io_adjustor_if_needed(Some(engines_info));
         }
+        let mut count = 0;
+        let tablet_registry = self.tablet_registry.clone().unwrap();
+        self.background_worker
+            .spawn_interval_task(Duration::from_secs(10), move || {
+                tablet_registry.for_each_opened_tablet(|_, cached| {
+                    if let Some(tablet) = cached.latest() {
+                        // rocksdb::kDefaultFlushInfoLogPeriodSec = 10.
+                        let _ = tablet
```

Comment:
Apparently flushing on certain size limit could cause foreground latency jitter. The periodic background flush is a safety net in case the existing manual flush sites (mostly during compaction and stuff) aren't called for a long time.

Reply:
Are we talking about info log or flushing memtables? How can it cause foreground latency? And what I mean is isn't log just written to file? Why need to flush?

Reply:
FWIW, rocksdb's native logger will buffer first before writing to a file. Even for direct pwrite there's the `fflush` to flush the OS page buffer.